### ISSUE TITLE
Long-term contributor looking to get more involved

### ISSUE THREAD

**Issue by jerryzh168** [pytorch/ao#3703]
Dear TorchAO (ao) team,

Hi, I'm Namgyu Youn (@namgyu-youn), who has been contributing to ao for a while (mostly Post-Training Quantization—Int8Tensor, AWQ/SmoothQuant, etc; https://github.com/pytorch/ao/pulls?q=is%3Apr+author%3Anamgyu-youn+)

<img width="456" height="212" alt="image" src="https://github.com/user-attachments/assets/fec0fa56-0d4f-4aed-9ab5-ee7855615d42" />

First, I want to thank Jerry for reviewing many of my PRs. The feedback has been really helpful for my learning. I wanted to reach out ao team because:
1) ao is closely aligned with my mid-term career direction
2) ao serves as a great guide to learn PyTorch internals like dispatch and layout
3) I prefer ao compared to other similar open-source projects

To be clear, I'm not asking to be promoted as a reviewer, triaged review, or to access internal infrastructure like team Slack. I understand those are for the core team, and my ability is quiet far from the team level.

What I'm hoping for is **merge permissions for approved PRs**. Some of my PRs sit approved but unmerged for a while (> 2 months). Would it be possible to get merge access just for PRs that have already been approved by maintainers?

Also, if there are any RFC discussions or areas where I could help with reviews (even as a learning opportunity), I'd be happy to participate—like AWQ/SmoothQuant, Int8Tensor, etc.—though I understand my ability is not yet at the team level.

Hope to grow with ao long-term, thanks!

**Comment by jerryzh168** [comment]
**Original author:** danielvegamyhre
**Posted at:** 2026-01-10T17:16:13Z

Thanks for all the great work you've done @namgyu-youn! cc @jerryzh168 @vkuzo to discuss

**Comment by jerryzh168** [comment]
**Original author:** vkuzo
**Posted at:** 2026-01-12T12:02:16Z

HI Namgyu, thank you for your contributions! A great channel for faster feedback is Slack - I added you on LinkedIn, feel free to message me your email and we can get you added.

**Comment by namgyu-youn** [comment]
@jerryzh168 following up on this discussion. Would "merge access for approved PRs" be possible? Label access (not by bot) and review requests would also be grateful to me.
